src/main/java/com/basho/riak/client/core/query/timeseries/FullColumnDescription.java

```diff
@@ -0,0 +1,138 @@
+package com.basho.riak.client.core.query.timeseries;
+
+/**
+ * Holds a complete definition for a Table Column in Time Series Riak.
+ * Immutable once created.
+ *
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.4
+ */
+public class FullColumnDescription
```

Comment:
But then we have a chicken and egg problem with the ColumnType Enum. We'd have to move it elsewhere (concrete enum referenced in interface), which would break anything that references it currently (external or internal).